.github/workflows/test-unit.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node: [12]
+        node: [14]
```

Comment:
Note that Remix needs Node 14 to install dependencies. Figured it's a good time to update usage in the tests, consider that Node 12 is EOL.